### PR TITLE
docs: JDK download url language from espanol to english

### PR DESCRIPTION
**Pull request by MrJithil**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/DEVELOPER.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ following products on your development machine:
 
 * [Yarn](https://yarnpkg.com) (version specified in the engines field of [`package.json`](../package.json)) which is used to install dependencies.
 
-* [Java Development Kit](http://www.oracle.com/technetwork/es/java/javase/downloads/index.html) which is used
+* [Java Development Kit](https://www.oracle.com/technetwork/es/java/javase/downloads/index.html) which is used
```
Comment (gkalpak) [inline docs/DEVELOPER.md]: Good catch. However, this is not actually changing the URL to not point to the spanish version :grin:
Can you please change the URL to `https://www.oracle.com/technetwork/java/javase/downloads/index.html`?

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: docs/DEVELOPER.md (hunk 1 above)
Comment (MrJithil) [reply docs/DEVELOPER.md]: thanks. I changed the URL from the language selection dropdown. But, it was not actually rebuilding the URL.

Edited.

**Comment by MrJithil** [comment]
@googlebot I signed it!

**Comment by MrJithil** [comment]
@googlebot I signed it!

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

**Pull request closed without merging**